src/audio-worklet/migration/worklet-recorder/app.js

```diff
@@ -310,3 +298,27 @@ function setupRecordingGainVis() {
 
   return draw;
 }
+
+/**
+ * Creating the downloadable .wav file for the recorded voice and set
+ * the download button clickable.
+ */
+function createRecord(recordingProperties, recordingLength, sampleRate, downloadLink, downloadButton, dataBuffer) {
+  const recordingBuffer = context.createBuffer(
+      recordingProperties.numberOfChannels,
+      recordingLength,
+      sampleRate);
+
+  for (let i = 0; i < recordingProperties.numberOfChannels; i++) {
+    recordingBuffer.copyToChannel(dataBuffer[i], i, 0);
+  }
+
+  const wavUrl = createLinkFromAudioBuffer(
+      recordingBuffer,
+      true);
+
+  player.src = wavUrl;
+  downloadLink.src = wavUrl;
+  downloadLink.download = 'recording.wav';
```

Comment:
You can add the timestamp so users can avoid multiple files with the same name. Something like:
```
'recording-' + new Date().getTime().toString().substr(-6) + '.wav'
```

You can be creative if you want! :)